doc/source/tutorial/match_columns.rst

```diff
@@ -290,6 +290,18 @@ On the other hand, the second one matches one article only because of ``Replies2
 Indexes with Weight
 -------------------
 
-TODO
+You can search blog entries which contains important keyword in ``Blog1`` table.
+For example, let's try to search blog entries which contains ``Groonga`` as important keyword.
+Generally speaking, an important keyword tend to be included in blog title, so if ``title`` contains ``Groonga``, its ``_score`` must be raised.
```

Comment:
In my opinion, we should clearly describe needing an index for using this feature.

Reply:
I got it.